dep/provider/dispatcher.go

```diff
@@ -0,0 +1,19 @@
+package provider
+
+import "github.com/short-d/kgs/app/usecase/event"
+
+type ListenerList struct {
+	Listeners []event.Listener
+}
+
+func BindListeners(dispatcher event.Dispatcher, list ListenerList) (event.Emitter, error) {
```

Comment:
No logic in dep. This should be handled in an use case.

Reply:
I will create a special helper function in `use case`